src/headers/tomcrypt_pk.h

```diff
@@ -292,13 +302,19 @@ int  ecc_test(void);
 void ecc_sizes(int *low, int *high);
 int  ecc_get_size(ecc_key *key);
 
+ltc_ecc_set_type* ecc_dp_find_by_oid(unsigned long *oid, unsigned long oidsize);
+ltc_ecc_set_type* ecc_dp_find_by_name(char *curve_name);
+ltc_ecc_set_type* ecc_dp_find_by_params(char *hex_prime, char *hex_A, char *hex_B, char *hex_order, char *hex_Gx, char *hex_Gy, unsigned long cofactor);
+
 int  ecc_make_key(prng_state *prng, int wprng, int keysize, ecc_key *key);
 int  ecc_make_key_ex(prng_state *prng, int wprng, ecc_key *key, const ltc_ecc_set_type *dp);
 void ecc_free(ecc_key *key);
 
 int  ecc_export(unsigned char *out, unsigned long *outlen, int type, ecc_key *key);
 int  ecc_import(const unsigned char *in, unsigned long inlen, ecc_key *key);
 int  ecc_import_ex(const unsigned char *in, unsigned long inlen, ecc_key *key, const ltc_ecc_set_type *dp);
+int  ecc_export_raw(unsigned char *out, unsigned long *outlen, int type, ecc_key *key);
```

Comment:
especially since this has the same signature as `ecc_export()`, how about re-using the same pattern that introduced `PK_STD`?
```
#define PK_RAW 0x2000
#define PK_COMPRESSED 0x4000
```